package.json

```diff
@@ -104,6 +104,7 @@
   },
   "dependencies": {
     "classnames": "^2.2.5",
+    "eventemitter3": "^4.0.4",
```

Comment:
Had to double check this wasn't a spoof package with a random number in there

Reply:
Yeah, the package name should have been a bit more creative.